requirements.txt

```diff
@@ -1,2 +1,3 @@
-python-telegram-bot==13.5
+python-telegram-bot
```

Comment:
`python-telegram-bot==20.6`

Reply:
in generale conviene impostare la versione in maniera fissa visto che verra' containerizzato, per questioni di comapatibilita' futura